autocrop/autocrop.py

```diff
@@ -204,8 +245,17 @@ def main(input_d,
         output_filename = os.path.join(output_d, basename)
         reject_filename = os.path.join(reject_d, basename)
 
+        # Open the image and detect the file type
+        img_orig = Image.open(input_filename)
```

Comment:
### Important
File I/O is typically the performance bottleneck in batch applications of autocrop. Since [OpenCV is usually an order of magnitude faster than PIL to read images](https://www.kaggle.com/vfdev5/pil-vs-opencv), it would make sense to use PIL only when reading image types OpenCV can't handle.

I know this means more code, but I don't think I'd merge this PR otherwise. Open to discussion , of course.